internal/config/config.go

```diff
@@ -55,9 +52,17 @@ func Load(path string) (*Config, error) {
 		}
 	}
 
-	// Decode
+	viper.SetConfigFile(path)
+	if err := viper.ReadInConfig(); err != nil {
+		return nil, err
+	}
+
+	viper.SetEnvPrefix("ize")
```

Comment:
I would set this to `IZE` (capitalized), I know viper is not case-sensitive, but since all env vars are capitalized usually, let's do it please